### ISSUE TITLE
iron-collapse should *not* set aria-expanded

### ISSUE THREAD

**Issue by TomHennen** [PolymerElements/iron-collapse#79]
### Description
iron-collapse incorrectly sets the aria-expanded attribute.

According to W3C aria-expanded should be set on the ["triggering link or button"](https://www.w3.org/WAI/GL/wiki/Using_aria-expanded_to_indicate_the_state_of_a_collapsible_element#Description).

iron-collapse is not the trigger link or button but rather the collapsible content itself.

Developers should set aria-expanded on the button that *triggers* the opening/closing of iron-collapse and it should *not* be set on iron-collapse itself.

### Expected outcome

When navigating into an iron-collapse region the screen reader should not indicate that the region is expanded.

### Actual outcome

When navigating into an expanded iron-collapse region the screen reader announces 'Expanded'. 

### Browsers Affected
I've only tested this on Chrome, but I suspect it applies everywhere a screen reader is used.
- [x] Chrome
- [ ] Firefox
- [ ] Safari 9
- [ ] Safari 8
- [ ] Safari 7
- [ ] Edge
- [ ] IE 11
- [ ] IE 10

**Comment by valdrinkoshi** [comment]
Ouch, true - we've been doing this wrong since https://github.com/PolymerElements/iron-collapse/pull/6
The fix should be trivial, just remove the setting of `aria-expanded` in tests and iron-collapse.
Btw we're setting this in the right node in our demo https://github.com/PolymerElements/iron-collapse/blob/master/demo/simple-expand-collapse.html#L48
It's probably worth updating the README.md example.
@TomHennen do you feel like sending a PR? 🍰 😃

**Comment by TomHennen** [comment]
Sure thing.  Might be next week.  Thanks!
